src/__tests__/opportunities.int.test.ts

```diff
@@ -133,4 +137,111 @@ describe('/opportunities', () => {
       });
     });
   });
+
+  describe('GET /id', () => {
```

Comment:
This test is just to add coverage to a feature added before, right?  Let's put it in a separate commit if that's easy (not a big deal if not)

Reply:
Correct, good call. Doing it.